resteasy-dependencies-bom/pom.xml

```diff
@@ -89,7 +89,7 @@
         <version.io.reactivex.rxjava>1.3.8</version.io.reactivex.rxjava>
         <version.io.reactivex.rxjava-reactive-streams>1.2.1</version.io.reactivex.rxjava-reactive-streams>
         <version.io.reactivex.rxjava2-rxjava>2.2.14</version.io.reactivex.rxjava2-rxjava>
-        <version.io.smallrye.smallrye-config>1.5.1</version.io.smallrye.smallrye-config>
+        <version.io.smallrye.smallrye-config>1.6.0</version.io.smallrye.smallrye-config>
```

Comment:
This can go to 1.6.1

Reply:
OK, will do, thanks

Reply:
I'm merging this and creating a proper jira+pr for the 1.6.1 upgrade